index.html

```diff
@@ -38,10 +38,10 @@
             <a href="#footer" id="navigation"><img src="images/reviews_logo_wht.svg" alt="reviews&more logo"
                     class="logo"></a>
             <div class="menu-right">
-                <div class="search-container"><input id="search" placeholder="Keresés" name="search" type="search"
-                        autocomplete="off" />
+                <div class="search-container"><label for="search"></label><input id="search" placeholder="Keresés"
+                        name="search" type="search" autocomplete="off" />
                 </div>
-                <button type="button" id="theme-btn" aria-label="Change to light theme" data-theme-toggle><span
+                <button type="button" id="theme-btn" aria-label="Change theme" data-theme-toggle><span
```

Comment:
Mivel az oldal magyarul van, az lenne konzisztens, ha az aria-label tartalma is magyar lenne.

Ez vonatkozik a masik html-ben hasznalt temavalto gombra is.